pywr/parameters/_parameters.pxd

```diff
@@ -78,6 +78,7 @@ cdef class TablesArrayParameter(IndexParameter):
     cdef public object where
 
     cdef int _scenario_index
+    cdef int[:] _scenario_ids
```

Comment:
Pedantic. I guess we should start commenting some of these attributes. I'm starting to forget what everything is!